### PR TITLE
Deletes logging sensitive information (fix for Issue #25)

### PR DESCRIPTION
**Pull request by ehaligow**
Deletes logging sensitive information (fix for Issue #25)
Signed-off-by: Haligowska, Emilia <emilia.haligowska@intel.com>

### CODE REVIEW

File: src/error_define.go [hunk 1]
```diff
@@ -131,6 +132,7 @@ func (e errorIndex) String(args ...string) string {
 		/*ErrUserStatus*/ "Please check this user account in this device",
 		/*ErrUserAdmin*/ "The user privilege is not administrator",
 		/*ErrUserPrivilege*/ "The user privilege could not configure this action to the device",
+		/*ErrWrongPrivilege*/ "Wrong privilege",
```
Comment (MaciejMis) [inline src/error_define.go:135]: Consider expanding error message - something like "Wrong privilege for this action"

File: src/config.go [hunk 2]
```diff
@@ -138,10 +138,7 @@ func (s *Server) getFunctionsResult(function string, deviceIPAddress string, aut
 			}
 		} else {
 			if errRet := s.validateDeviceAccountData(deviceIPAddress, userName, password); errRet != "" {
-				logrus.WithFields(logrus.Fields{
-					"IP address:port": deviceIPAddress,
-					"Username":        userName,
-				}).Errorf(errRet)
+				logrus.Errorf(errRet)
```
Comment (MaciejMis) [inline src/config.go:141]: More targeted error message would be good here (something similar to other error messages in this PR)

Comment (ehaligow) [reply src/config.go:141]: The value "errRet" already has more targeted message - it is assigned in validateDeviceAccountData function.

File: src/error_define.go (hunk 1 above)
Comment (ehaligow) [reply src/error_define.go]: Error message changed.

**Pull request merged**